File: 프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.py
def solution(my_string, is_prefix):
    answer = 0
    if(len(my_string) >= len(is_prefix)):
        if(my_string[0] == is_prefix[0]):
            for i in range(len(is_prefix)):
                if(my_string[i] == is_prefix[i]):
                    answer = 1
                else:
                    answer = 0
                    break
        else:
            answer = 0
    else:
        answer = 0
    return answer

File: 프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.js
function solution(my_string, is_prefix) {
    var answer = 0;
    if(my_string.length < is_prefix.length) return 0;
    if(my_string[0] == is_prefix[0]){
        for(let i = 0; i < is_prefix.length; i++){
            if(my_string[i] == is_prefix[i]){
                answer = 1;
            }
            else{
                answer = 0
            }
        }
    }
    return answer;
}